config/sim_config.lua

```diff
@@ -43,9 +47,11 @@ angular_error_rate = 0.1;
 -- Defining robot type enumerator
 local RobotType = {
     F1TEN=0,
-    COBOT=1
+    COBOT=1, 
+    BWIBOT=2
 }
 
 -- robot_type = RobotType.F1TEN
 robot_type = RobotType.COBOT
+-- robot_type = RobotType.BWIBOT
```

Comment:
This comment does seem to be put in a right place. Probably after the laser topic?

Reply:
I was thinking to just group the `robot_types` together? Though I agree the `laser_topic` should immediately follow the cobot spec